### PR TITLE
Rutracker-ru add russian option

### PR DESCRIPTION
**Pull request by alikhanz**
Second public indexer must have addrussian option.

https://github.com/Jackett/Jackett/issues/14021

### CODE REVIEW

**Comment by bakerboy448** [comment]
Won't this mess up groups? `-Group` => `-GroupRussian` ?

File: src/Jackett.Common/Definitions/rutracker-ru.yml [hunk 1]
```diff
@@ -389,6 +389,9 @@ settings:
     type: checkbox
     label: Strip Russian Letters
     default: false
+  - name: addrussian
+    type: checkbox
+    label: Add RUSSIAN to end of all titles to improve language detection by Sonarr and Radarr. Will cause English-only results to be misidentified.
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/rutracker-ru.yml:394]:
```suggestion
    label: Add RUS to end of all titles to improve language detection by Sonarr and Radarr. Will cause English-only results to be misidentified.
```

File: src/Jackett.Common/Definitions/rutracker-ru.yml [hunk 2]
```diff
@@ -510,6 +513,8 @@ search:
           args: ["WEBDLRip", "WEBDL"]
         - name: replace
           args: ["HDTVRip", "HDTV"]
+        - name: append
+          args: "{{ if .Config.addrussian }} - RUSSIAN{{ else }}{{ end }}"
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/rutracker-ru.yml:517]:
```suggestion
          args: "{{ if .Config.addrussian }} RUS {{ else }}{{ end }}"
```

**Review by garfield69 — CHANGES_REQUESTED** [review]

**Comment by alikhanz** [comment]
> Won't this mess up groups? `-Group` => `-GroupRussian` ?

I have no issues with other indexers with same logic

**Comment by alikhanz** [comment]
@garfield69 can we merge this PR?

**Comment by ilike2burnthing** [comment]
https://github.com/Jackett/Jackett/blob/fc79f23d323f4fb2d9dcb15393494eccd0fe47fd/src/Jackett.Common/Definitions/rutracker-ru.yml#L489-L502

We're adding `rus` when seasons and episodes are re_replaced. Either that would need to be removed, or we just leave it hardcoded for TV results only. The same thing is done in a bunch of other Russian trackers as well.

**Comment by alikhanz** [comment]
> https://github.com/Jackett/Jackett/blob/fc79f23d323f4fb2d9dcb15393494eccd0fe47fd/src/Jackett.Common/Definitions/rutracker-ru.yml#L489-L502
> 
> We're adding `rus` when seasons and episodes are re_replaced. Either that would need to be removed, or we just leave it hardcoded for TV results only. The same thing is done in a bunch of other Russian trackers as well.

I think delete this logic for tv shows would be right solution

**Comment by alikhanz** [comment]
@ilike2burnthing can we merge?

**Comment by ilike2burnthing** [comment]
Going to take a closer look at the re_replaces, see if there's a better way to do them, especially if we'd end up rolling this out to other indexers.

**Comment by alikhanz** [comment]
> Going to take a closer look at the re_replaces, see if there's a better way to do them, especially if we'd end up rolling this out to other indexers.

lgtm
tbh im newbie in radarr/sonarrr+jackett stack, so idk how i can improve this filter. So, we can wait someone like @mynameisbogdan for approving this pr maybe?

**Comment by ilike2burnthing** [comment]
Will look at other indexers tomorrow.

**Pull request merged**